package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "whitelist-smart-contract",
+  "version": "0.0.1",
+  "description": "Whitelist smart contract to allow developers to not resign a sell order every time there's a new web3mail dapp being deployed.",
+  "main": "index.ts",
+  "scripts": {
+    "compile": "hardhat compile",
+    "test": "hardhat test",
+    "coverage": "hardhat coverage",
+    "deploy": "hardhat run scripts/deploy.ts --network bellecour",
+    "addRessourceToWhitelist": "hardhat run scripts/addRessourceToWhitelist.ts --network bellecour",
+    "removeRessourceFromWhitelist": "hardhat run scripts/removeRessourceFromWhitelist.ts --network bellecour",
+    "transferOwnership": "hardhat run scripts/transferOwnership.ts --network bellecour",
+    "verify": "hardhat verify",
+    "clean": "hardhat clean",
+    "lint": "eslint **/*.ts",
+    "lint:fix": "eslint **/*.ts --fix",
+    "typecheck": "tsc --noEmit",
+    "format": "prettier --plugin=./node_modules/prettier-plugin-solidity/src/index.js --write \"{scripts,test,contracts}/**/*.{ts,sol}\"",
+    "check-format": "prettier --plugin=./node_modules/prettier-plugin-solidity/src/index.js --check \"{scripts,test,contracts}/**/*.{ts,sol}\""
+  },
```

```suggestion
  "scripts": {
    "compile": "hardhat compile",
    "test": "hardhat test",
    "coverage": "hardhat coverage",
    "deploy": "hardhat run scripts/deploy.ts --network bellecour",
    "addRessourceToWhitelist": "hardhat run scripts/addRessourceToWhitelist.ts --network bellecour",
    "removeRessourceFromWhitelist": "hardhat run scripts/removeRessourceFromWhitelist.ts --network bellecour",
    "transferOwnership": "hardhat run scripts/transferOwnership.ts --network bellecour",
    "verify": "hardhat verify",
    "clean": "hardhat clean",
    "lint": "eslint . --ext .ts",
    "format": "prettier --plugin=./node_modules/prettier-plugin-solidity/src/index.js --write \"{scripts,test,contracts}/**/*.{ts,sol}\"",
    "check-format": "prettier --plugin=./node_modules/prettier-plugin-solidity/src/index.js --check \"{scripts,test,contracts}/**/*.{ts,sol}\""
  },
```
remove useless scripts (lint:fix + typecheck)